hasura/graphql-engine
# ra-data-hasura: fix column sorting

### Description
Column sorting was not working. All column header clicks sorted by `primaryKey`/`id`.

### Affected components 
- ra-hasura-admin

### Solution and Design
Set sort column to `params.sort.field` and default to `primaryKey` just in case.

### Steps to test and verify
Try to sort a column, click on any column header and notice they all perform the same sort action. With this change the columns will be sorted properly.
